### PR TITLE
New version: Tencent.QQMusic version 20.47

### LINKED ISSUE #198553: [Update Request]: Tencent.QQMusic: 20.47
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Tencent.QQMusic

### Package Version

20.47

### Please describe the changes you would like to see

There is a new release (20.47) available.
Download Page Url: https://dldir1v6.qq.com/music/clntupate/QQMusic_Setup_2047.exe

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.8.0 :rocket:

- Resolves #198553
- Use the following command to update it:
  ```powershell
  komac update Tencent.QQMusic --version 20.47 --urls https://dldir1v6.qq.com/music/clntupate/QQMusic_Setup_2047.exe
  ```
- I'm not sure if the previous package URL is still available now...

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/198679)

### CODE REVIEW

File: manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- 歌单
+- 歌手
+- 歌曲
+- 音乐
+- 音频
+ReleaseNotes: |-
+  1. 播放器功能优化，新增浅色样式
+  2. 播放条功能优化，支持歌词显示与调节高度
+  3. 播放器内右键菜单新增歌词居中功能
+  4. 其他体验优化
```
Comment (SpecterShell) [inline manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml:32]:
```suggestion
  1. 臻品母带与 SQ 音质支持省流模式，开启后可节省更多流量
  2. 其他体验优化
```

Comment (DuckDuckStudio) [reply manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml]: komac seems to have uses the release notes directly from the last version...

File: manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PackageIdentifier: Tencent.QQMusic
+PackageVersion: '20.47'
+PackageLocale: zh-CN
+Publisher: 腾讯科技（深圳）有限公司
+PublisherUrl: https://y.qq.com/
+PublisherSupportUrl: https://y.qq.com/m/client/helper/pc.html
+PrivacyUrl: https://y.qq.com/m/client/intro/privacy.html
+Author: 腾讯音乐娱乐科技（深圳）有限公司
+PackageName: QQ音乐
+#PackageUrl: https://y.qq.com/download/welcome_pc_v15/index.html
```
Comment (Dragon1573) [inline manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml:13]:
```suggestion
PackageUrl: https://y.qq.com/download/download.html
```

File: manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+PackageIdentifier: Tencent.QQMusic
+PackageVersion: '20.47'
+PackageLocale: en-US
+Publisher: 腾讯科技（深圳）有限公司
+PublisherUrl: https://y.qq.com/
+PublisherSupportUrl: https://y.qq.com/m/client/helper/pc.html
+PrivacyUrl: https://y.qq.com/m/client/intro/privacy.html
+Author: Tencent Music Entertainment Technology (Shenzhen) Co., Ltd.
+PackageName: QQ Music
+#PackageUrl: https://y.qq.com/download/welcome_pc_v15/index.html
```
Comment (Dragon1573) [inline manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://y.qq.com/download/download.html
```

**Review by Dragon1573 — COMMENTED** [review]
Keep `PackageUrl` in the manifest for consistency might be better.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**